packages/space-opera/src/test/materials_panel/materials_panel_test.ts

```diff
@@ -74,10 +75,6 @@ describe('material panel test', () => {
     await dispatchGltfAndEdits(gltf);
   });
 
-  afterEach(() => {
-    document.body.removeChild(panel);
```

Comment:
Maybe leave this commented out with a comment about the surprising problems that came with it. Maybe mark as a TODO, since it probably should be here.